src/axom/inlet/DocWriter.hpp

```diff
@@ -0,0 +1,35 @@
+#include <string>
+#include <vector>
+#include <fstream>
+
+#include "axom/sidre.hpp"
+
+#ifndef INLET_DOCWRITER_HPP
+#define INLET_DOCWRITER_HPP
+
+namespace axom
+{
+namespace inlet
+{ 
+
+class DocWriter {
+public: 
+  DocWriter(const std::string& fileName, axom::sidre::Group* sidreRootGroup, bool isVerbose = true);
+  void writeTitle(const std::string& title);

```

Comment:
These three write classes can be moved to private since the user shouldn't be calling them.